JourneysSquad/JourneysSquad/Model/DummyBillModel.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+/// A statement of the money owed for goods or services by different people.
+struct DummyBillModel {
+
+    /// The description of the bill.
+    let description = "Description"
+
+    /// Participants who is on the account.
+    let participants = ["Participant 1", "Participant 2"]
+
+    /// The overall sum of money that was spent within the bill.
+    let sum = "50.00 $"
```

Comment:
```suggestion
    let sumOfBill = "50.00 $"
```
 It will express more certainly the meaning of the constant.

Reply:
fixed